node/service/Cargo.toml

```diff
@@ -9,8 +9,7 @@ edition.workspace = true
 # Substrate Client
 sc-authority-discovery = { git = "https://github.com/paritytech/substrate", branch = "master" }
 babe = { package = "sc-consensus-babe", git = "https://github.com/paritytech/substrate", branch = "master" }
-beefy-primitives = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-beefy" }
-beefy-gadget = { git = "https://github.com/paritytech/substrate", branch = "master" }
+beefy-gadget = { package = "sc-consensus-beefy", git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
I would also change this (and the code references) to `sc-consensus-beefy` or `sc-beefy`;

Same for `beefy-primitives` -> `sp-beefy`